astropy/tests/pytest_plugins.py

```diff
@@ -462,17 +462,31 @@ def pytest_runtest_teardown(item, nextitem):
                                      ('Matplotlib', 'matplotlib'),
                                      ('h5py', 'h5py')])
 
+# This always returns with Astropy's version
+from .. import __version__
+
+TESTED_VERSIONS = OrderedDict([('Astropy', __version__)])
```

Comment:
It's not entirely clear to me how this is intended to be used, but it's definitely a good thing to fix in principle.